scripts/us_eia/opendata/process/common.py

```diff
@@ -89,10 +95,28 @@ def _find_dc_place(raw_place, is_us_place, counters):
         if raw_place in alpha2_to_dcid.USSTATE_MAP:
             return alpha2_to_dcid.USSTATE_MAP[raw_place]
     else:
-        if raw_place in alpha2_to_dcid.COUNTRY_MAP:
+        if len(raw_place) == 2 and raw_place in alpha2_to_dcid.COUNTRY_MAP:
             return alpha2_to_dcid.COUNTRY_MAP[raw_place]
+        elif len(raw_place) == 3 and raw_place in _ALPHA3_COUNTRY_SET:
+            return f'country/{raw_place}'
+        elif len(raw_place) > 3:
+            # INTL dataset has 40 country aggregates
+            # (http://screenshot/8YvH4w58iDtuc5r).  We map a subset that exists
```

Comment:
maybe find a way for github to host this?  this is a bit of a hack but would work:
https://medium.com/@justynagolawska/how-to-easily-add-screenshots-into-your-readme-file-on-github-d806a01d6ffd

Reply:
good catch. nice find.